api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -134,7 +134,7 @@ public List<FileDetail> getAllNotebooksByAppType(
     if (appType.equals(AppType.RSTUDIO)) {
       return allNotebooks.stream()
           .filter(fileDetail -> NotebookUtils.isRStudioFile(fileDetail.getName()))
-          .collect(Collectors.toList());
+          .toList();
```

Comment:
the API and UI change are just to ensure test runs.  I'll remove these before merging.